docs/core/tools/global-json.md

```diff
@@ -1,32 +1,104 @@
 ---
-title: global.json reference
-description: See the schema for the global.json file, which permits setting the .NET Core tools version.
-author: blackdwarf
+title: global.json overview
+description: Learn how to use the global.json file to set the .NET Core SDK version when running .NET Core CLI commands.
+author: mairaw
 ms.author: mairaw
-ms.date: 04/05/2017
+ms.date: 07/02/2018
+ms.custom: "updateeachrelease"
```

Comment:
Should this also have "mvc" with a topic type of "conceptual"?

Reply:
I'm not sure how you all do topics, but...

This is relevant for all .NET Core apps, there is nothing about MVC that makes it more or less relevant, so I would suggest not including it, or including all application types that work in .NET Core.

Reply:
@KathleenDollard The "MVC" moniker here is an internal APEX term that is being changed because of confusion with the "ASP.NET MVC" product. The two are unrelated.

Reply:
I thought that conceptual wasn't really a mvc topic type @JRAlexander.

Reply:
As far as I know, it is.